README.md

```diff
@@ -20,3 +20,21 @@ Sometimes developers need an embedded HTTP server in their app. Perhaps it's a s
 -   Heavily commented code
 -   Very easily extensible
 -   WebDAV is supported too!
+
+## Getting Started
+
+If you’re using [Carthage](https://github.com/Carthage/Carthage), simply add LivioHTTPServer to your `Cartfile`:
+
+```
+github "PhysicalGraph/LivioHTTPServer"
+```
+Make sure to add `CocoaAsyncSocket.framework` to "Linked Frameworks and Libraries" and "copy-frameworks" Build Phases.
```

Comment:
Should include a link to https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos